src/effects/backends/builtin/autogaincontroleffect.cpp

```diff
@@ -0,0 +1,241 @@
+#include "effects/backends/builtin/autogaincontroleffect.h"
+
+#include "util/math.h"
+
+namespace {
+constexpr double defaultAttackMs = 1;
+constexpr double defaultReleaseMs = 500;
+constexpr double defaultThresholdDB = -40;
+constexpr double defaultTargetDB = -5;
+constexpr double defaultGainDB = 20;
+constexpr double defaultKneeDB = 10;
```

Comment:
```suggestion
constexpr double kDefaultAttackMs = 1;
constexpr double kDefaultReleaseMs = 500;
constexpr double kDefaultThresholdDB = -40;
constexpr double kDefaultTargetDB = -5;
constexpr double kDefaultGainDB = 20;
constexpr double kDefaultKneeDB = 10;
```
see: https://github.com/mixxxdj/mixxx/wiki/Coding%20Guidelines#constexpr

consider to use time_literals: https://en.cppreference.com/w/cpp/chrono/operator%2522%2522ms.html

Reply:
Fixed.

Regarding time literals: attack and release parameters could be fractional. So I changed to time literals only these 2 default values.